packaging/resource_suite_s3_cache.py

```diff
@@ -105,6 +105,9 @@ def setUp(self):
 
         s3params=os.environ.get('S3PARAMS', s3params);
 
+        self.archive_resc = 's3'
+        self.cache_resc = 'cacheResc'
```

Comment:
What are these for? I don't see them being referenced in the file anywhere.

Reply:
Ah, some leftovers. Will remove. Had some big dreams for some new `lib.py` functions and decided against it for the sake of expediency :)

Reply:
Got it. Looking forward to those enhancements for lib.py.

Reply:
Done